gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationViewModel.java

```diff
@@ -360,6 +360,10 @@ private void onPhotoResult(Bitmap bitmap) {
     lastPhotoResult.onNext(PhotoResult.create(fieldId, Optional.of(bitmap)));
   }
 
+  public void clearPhoto(String fieldId) {
```

Comment:
@shobhitagarwal1612 Not sure why but this is never called when tapping the trash icon.

Reply:
The top-level was PhotoInputFieldBinding, not PhotoFieldBinding, and I wasn't passing the view model down through the layout vars. Fixed!